docs/source/tutorials/ordinary_differential_equations.rst

```diff
@@ -13,6 +13,7 @@ See the notebooks below to learn more about how to use ProbNum's functionality f
 
    adaptive_steps_odefilter
    uncertainties_odefilters
+   odesolvers_from_scratch
```

Comment:
I worked in the non-`probsolve_ivp` universe to verify my implementation and decided to just glue the code into a notebook. Nothing fancy, just an illustration of how to solve ODEs without `probsolve_ivp`.